examples/webgl_loader_svg.html

```diff
@@ -193,7 +194,7 @@
 							const material = new THREE.MeshBasicMaterial( {
 								color: new THREE.Color().setStyle( strokeColor ),
 								opacity: path.userData.style.strokeOpacity,
-								transparent: path.userData.style.strokeOpacity < 1,
+								transparent: true,
```

Comment:
I doubt users will understand this subtlety.  Doing so requires knowledge of the inner-workings of the renderer...

Reply:
But do you think they'll notice any change?

Reply:
If setting non-transparent objects to `transparent` is required for three.js to sort and render `svg` objects correctly, then what we have here is a three.js limitation -- or bug.

The work-around in this example needs to be explained with some comments, no?

Reply:
I don't think it's particularly a three.js limitation - SVG isn't a 3D format and so a best-effort attempt at achieving SVG semantics in a 3D engine involves ensuring the individual meshes are all `depthWrite: false` (to avoid depth fighting) and rendered in order regardless of their opacity; which in three.js implies setting all of them `transparent`.

I'd argue that `transparent: true` is likely the right choice whenever materials are set `depthWrite: false` as the ability to arbitrarily order draw calls without affecting the output relies on both `depthTest` and `depthWrite` being used.

I'd be happy for a comment to be added (perhaps applying to both the `depthWrite` and `transparent` lines for the opaque meshes) that either points here or to #22898 if you guys think it helps understanding. 

Potentially three.js could also use some better way of ensuring ordering among a particular set of meshes (perhaps via using a single BufferGeometry with groups) but then it's hard to know where how that whole group should sort with respect to everything else in the scene.

Reply:
Your explanation in the main post was excellent. I'd be happy with a comment in the code if @mrdoob and @Mugen87 agree.

Reply:
I would definitely link to this discussion. A simple addition of `// #22912` should be sufficient to me.

Reply:
>A simple addition of // #22912 should be sufficient.

That would be fine for source code. But for an _example_, I think a short explanation is warranted,

Reply:
I suggest to add the comment where `transparent: true` is set in the example. Of course an additional explanation is fine.